stackstorm/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM debian:wheezy
 ARG ST2_VERSION
 LABEL com.stackstorm.version="${ST2_VERSION}"
 
-RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2
+RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev
```

Comment:
what actual version of libffi we need. I guess `libffi5` and `libffi-dev` are inconsistent...